src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx

```diff
@@ -128,6 +128,16 @@ export default function EnvTriggerView({ filteredAppIds, isVirtualEnv }: AppGrou
     const [isSelectAll, setSelectAll] = useState(false)
     const [selectAllValue, setSelectAllValue] = useState<CHECKBOX_VALUE>(CHECKBOX_VALUE.CHECKED)
     const abortControllerRef = useRef(new AbortController())
+    const [appReleaseTags, setAppReleaseTags] = useState<string[]>([])
```

Comment:
why do we need these 2 states. It can be handled via workflow data

Reply:
we need these states as these will be updated in the child component